src/management-system-v2/lib/process-export/export-preparation.ts

```diff
@@ -105,10 +125,9 @@ type ExportMap = {
     definitionName: string;
     versions: {
       [version: string]: {
+        name?: string;
         bpmn: string;
-        artefactsToExport: {
-          userTaskFiles: string[];
-        };
+        subprocesses: { id: string; name: string }[];
```

Comment:
See my comment in [Line 42](https://github.com/PROCEED-Labs/proceed/pull/122/files#diff-1b618ca2580a9520819efd507148b3d8d3be83aedfeedf07ecb606fe8579d5eaR42)